src/main/resources/org/sonar/l10n/authgithub.properties

```diff
@@ -1,5 +1,5 @@
 property.category.github=GitHub
 property.category.github.authentication=Authentication
-property.category.github.authentication.description=In order to enable GitHub authentication:<ul><li>SonarQube must be publicly accessible through HTTPS only</li><li>The property 'sonar.core.serverBaseURL' must be set to this public HTTPS URL</li><li>In your GitHub profile, you need to create a Developer Application for which the 'Authorization callback URL' must be set to <code>'&lt;value_of_sonar.core.serverBaseURL_property&gt;/oauth2/callback'</code>.</li></ul>
+property.category.github.authentication.description=In order to enable GitHub authentication:<ul><li>SonarQube must be publicly accessible through HTTPS only</li><li>The property 'sonar.core.serverBaseURL' must be set to this public HTTPS URL</li><li>In your GitHub profile, you need to create a Developer Application for which the 'Authorization callback URL' must be set to <code>'&lt;value_of_sonar.core.serverBaseURL_property&gt;/oauth/callback/github'</code>.</li></ul>
```

Comment:
The url to be used must really match "/oauth2/callback" and not "/oauth2/callback/github".
Even if "/oauth2/callback/github" could indeed work, but it's a technical detail and it's not needed.
Please test again and send a message to [community.sonarsource.com](community.sonarsource.com) if it's failing.

Reply:
Thanks for the quick reply, error was on my part indeed. Was testing changes to the callback URL, but in the meantime a colleague had updated the config to use a different client ID.

What ultimately caused the confusion, if you specify the base URL property (sonar.core.serverBaseURL) with a trailing slash, it leads to the following error:

`ERROR web[AWQYLJFv1CTcIbTDAATo][o.s.s.a.AuthenticationError] No provider key found in URI`

Will close the PR.